swiper.el

```diff
@@ -550,7 +589,8 @@ numbers; replaces calculating the width from buffer line count."
               (let ((str (swiper--line)))
                 (setq str (ivy-cleanup-string str))
                 (let ((line-number-str
-                       (format swiper--format-spec line-number)))
+                       (propertize (format swiper--format-spec line-number)
+                                   'face 'swiper-line-number-face)))
```

Comment:
You could probably save a string allocation here by using `put-text-property` instead of `propertize`, right?